src/frameworks/openai/openaiconverter.ts

```
@@ -0,0 +1,158 @@
+          case 'web_search_call':
+            messages.push({
+              role: "assistant" as const,
+              tool_calls: [{
+              name: outputElement.name,
+                arguments: {},
+                id: outputElement.id,
+              }],
+            });
+            break;
+          case 'file_search_call':
+            let toolArguments = outputElement.queries? {"queries": outputElement?.queries} : {};
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)